hs_file_types/tests/test_geofeature_metadata_user_zone.py

```diff
@@ -0,0 +1,102 @@
+from django.test import TransactionTestCase
+from django.contrib.auth.models import Group
+from django.conf import settings
+
+from hs_core import hydroshare
+from hs_core.hydroshare.utils import resource_post_create_actions
+from hs_core.testing import TestCaseCommonUtilities
+from utils import assert_geofeature_file_type_metadata
+from hs_file_types.models import GeoFeatureLogicalFile, GenericLogicalFile, GeoFeatureFileMetaData
+
+
+class GeoFeatureFileTypeMetaDataTest(TestCaseCommonUtilities, TransactionTestCase):
+    def setUp(self):
+        super(GeoFeatureFileTypeMetaDataTest, self).setUp()
+        if not super(GeoFeatureFileTypeMetaDataTest, self).is_federated_irods_available():
+            return
+
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        self.user = hydroshare.create_account(
+            'user1@nowhere.com',
+            username='user1',
+            first_name='Creator_FirstName',
+            last_name='Creator_LastName',
+            superuser=False,
+            groups=[self.group]
+        )
+
+        super(GeoFeatureFileTypeMetaDataTest, self).create_irods_user_in_user_zone()
+
+        base_file_path = 'hs_file_types/tests/data/{}'
+        self.zip_file_name = 'states_required_files.zip'
+        self.zip_file = base_file_path.format(self.zip_file_name)
+
+        # transfer this valid tif file to user zone space for testing
+        # only need to test that tif file stored in iRODS user zone space can be used to create a
+        # composite resource and metadata can be extracted when the file type is set to raster file
+        # type.
+        # Other detailed tests don't need to be retested for irods user zone space scenario since
+        # as long as the tif file in iRODS user zone space can be read with metadata extracted
+        # correctly, other functionalities are done with the same common functions regardless of
+        # where the tif file comes from, either from local disk or from a federated user zone
```

Comment:
@pkdash This is probably a copy and paste issue since the comment above does not appear to apply here.